Graphite/GraphiteClient.cs

```diff
@@ -146,11 +146,12 @@ public async Task<string[]> ExpandMetricsAsync(bool leavesOnly, CancellationToke
         /// <param name="until">specify the relative or absolute end to graph</param>
         /// <param name="template">The target metrics can use a special <see cref="SeriesListBase.Template(string[])"/> function which allows the metric paths to contain variables</param>
         /// <param name="maxDataPoints"></param>
+        /// <param name="tz">specify time zone for from/until. Corresponds to 'tz' parameter in Graphite Render API.</param>

```

Comment:
Can we use the description from the graphite docs?
> Time zone to convert all times into.

Reply:
Good idea! Fixed.